scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -989,6 +989,28 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) { parser =>
     case _ => false
   }
 
+  def followedByToken[T <: Token: TokenInfo]: Boolean = {
+    def startBlock = token.is[LeftBrace] || token.is[Indentation.Indent]
+    def endBlock = token.is[RightBrace] || token.is[Indentation.Outdent]
```

Comment:
should it skip parens too?

Reply:
It doesn't in dotty, so I would rather keep it the same as the Scala 3 implementation.